extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1080,7 +1081,12 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     element.classList.add('i-amphtml-lbg-gallery-thumbnail');
     const imgElement = this.win.document.createElement('img');
     imgElement.classList.add('i-amphtml-lbg-gallery-thumbnail-img');
-    imgElement.setAttribute('src', thumbnailObj.url);
+
+    if (thumbnailObj.srcset) {
+      imgElement.setAttribute('srcset', thumbnailObj.srcset.stringify());
```

Comment:
TODO: pending `sizes` and `srcset` fix, we can revisit the logic here to make sure this works properly with them.

Reply:
I think if both `src` and `srcset` are present on an image, both are semantically meaningful. I think here you want to apply each to `imgElement`, not just either one. Here's an example of both being used in the spec: http://w3c.github.io/html/semantics-embedded-content.html#example-15076cde

Reply:
The intention here is that if the user provided a `srcset` or we can generate a `srcset` from any kind of placeholder, then we use `srcset`. However, if the user did not provide a `srcset`, then we use our default placeholders. I originally wanted to only use `srcset`, however data uri with svgs don't work with `srcset`. Hence if a `srcset` is ever provided, we don't want to be using `src`. Maybe it is more appropriate if I put a comment here.

Reply:
Oh I see, `srcsetFromElement` converts the `src` to `srcset` if needed. 👍